Create cde-sex-to-odm.xsparql

sxzhang1201 opened the pull request.

sxzhang1201 committed:
Create cde-sex-to-odm.xsparql

rajaram5 commented on cde-to-odm/xsparql/cde-sex-to-odm.xsparql:
```diff
@@ -0,0 +1,48 @@
+                               $output rdf:type sio:SIO_000340.
+                               $output rdf:type $sex.
+                             }
+                             union
```
Why do we need `union` here? `SPARQL` should able to handle graph pattern belongs to same graph

sxzhang1201 replied:
As there are two pathways to reach the "sex" quality in the CDE model, I choose UNION to represent both pathways. But it might be a next step to think about :)

rajaram5 replied:
ah ok, but still you can do it with `UNION` right. Because according to CDE ShEx both these paths should exist in a valid CDE RDF.

rajaram5 commented on cde-to-odm/xsparql/cde-sex-to-odm.xsparql:
```diff
@@ -0,0 +1,48 @@
+                               $output rdf:type $sex.
+                             }
+
+                            filter(strstarts(str($sex), "http://purl.bioontology.org/ontology/SNOMEDCT/"))
```
why do we need this filter?

rajaram5 commented on cde-to-odm/xsparql/cde-sex-to-odm.xsparql:
```diff
@@ -0,0 +1,48 @@
+
+                            filter(strstarts(str($sex), "http://purl.bioontology.org/ontology/SNOMEDCT/"))
+                           }
+                       return <ItemDataURI ItemOID="SEX">{$sex}</ItemDataURI>
```
don't we have to convert value of sex from IRI to text (M or F)?

sxzhang1201 replied:
You are right! I will fix it.

sxzhang1201 replied:
You are right! I will fix it.

sxzhang1201 committed:
Update of XSPARQL

Two XSPARQL have different XML structures, both of them will be updated after discussion.

sxzhang1201 committed:
Create README.md for cde-to-odm

sxzhang1201 committed:
Correct Sytactical Error
